### PR TITLE
Ignore repositories[#287]

### PR DESCRIPTION
**Pull request by amitk**

### CODE REVIEW

File: app/models/activities_fetcher.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def fetch(type = :daily)
       if TRACKING_EVENTS.key?(a.type) && Time.parse(a.created_at) > since_time
         repo = Repository.where(gh_id: a.repo.id).first
         repo = self.create_repo(a.repo.name) unless repo
-        create_activity(a, repo) if repo
+        create_activity(a, repo) if repo && repo.ignore == false
```
Comment (prasadsurase) [inline app/models/activities_fetcher.rb]: @amitk the condition can be rewritten as "if repo && !repo.ignore"

File: app/models/repository.rb [hunk 2]
```diff
@@ -18,6 +18,7 @@ class Repository
   field :ssh_url,      type: String
   field :ignore_files, type: Array, default: []
   field :type,         type: String
+  field :ignore, type: Boolean, default: false
```
Comment (prasadsurase) [inline app/models/repository.rb]: indent it as per the above code.

File: app/views/admin/repositories/index.html.haml [hunk 3]
```diff
@@ -1,3 +1,9 @@
+%section.content-header
+  %h1
+    Repositories:-
+    %label.slide
+      %input#repo{ :checked => "checked", :type => "checkbox", data: {toggle: 'toggle', width: 100, height: 40, on: "Required", off: "Ignored"}}
```
Comment (prasadsurase) [inline app/views/admin/repositories/index.html.haml]: you can replace the '=>' with ':'

**Comment by amitk** [comment]
closed due to unrelated commits.

**Pull request closed without merging**